src/proto/h1/role.rs

```diff
@@ -77,7 +77,7 @@ where
         return Ok(None);
     }
 
-    let _entered = trace_span!("parse_headers");
+    trace_span!("parse_headers");
```

Comment:
I think this changes the functionality.

Reply:
Yes, This is a Clippy false positive warnings.